service/remote_processor_service.py

```diff
@@ -90,19 +90,36 @@ def ProcessResponse(self, request: ProcessResponseRequest, context):
             context.set_code(grpc.StatusCode.NOT_FOUND)
             context.set_details(f'Processor {processor_name} does not exist!')
             raise KeyError(f"Processor {processor_name} does not exist! Check your config file {self._config_file}")
-        
-    def start(self):
+
+    def start(self, certfile=None, keyfile=None):
         """Start the server on port 2796 (ARYN on a keypad)
 
         Returns:
             Server: a grpc server object
         """
         logging.info(PAPRIKA_ASCII_ART)
+        secure_mode = False
+        if certfile is not None and keyfile is not None:
+            secure_mode = True
+            logging.info("Starting service in secure mode")
+        else:
+            logging.info("Starting service in insecure mode")
```

Comment:
You could literally have: 'secure_mode = bool(certfile and keyfile)' because in this case empty strings aren't valid.  You should move the logging down below inside the 'if secure_mode'.

Reply:
fair